data/json/statistics.json

```diff
@@ -751,7 +751,7 @@
     "type": "event_statistic",
     "stat_type": "count",
     "event_transformation": "avatar_gains_barter_level_7",
-    "description": { "str_sp": "bartering skill level 7 gained" }
+    "description": { "str_sp": "social skill level 7 gained" }
```

Comment:
This might better keep the original name becasue `barter` is obsolete instead of changed to `social` skill. (Is there a value to mark achievement as obsolete?)

Reply:
Reverted to bartering in both cases

Reply:
If there's a process for obsoleting achievements, please let me know.

Reply:
We can simply delete obsolete achievements and that shouldn't cause any bugs, but it probably  would mean users wouldn't see historical gains of that achievement any more.  If we introduced a way to obsolete them then we could make that work better.

In short, don't worry about it for now.